README.md

```diff
@@ -48,22 +48,22 @@ Installation
 ------------
 
 Install the plugin package from either the yum or the apt repository.
+To setup those package repositories, see documentation about installation of mackerel-agent ([rpm](https://mackerel.io/docs/entry/howto/install-agent/rpm) / [deb](https://mackerel.io/docs/entry/howto/install-agent/deb)).
```

Comment:
[nit] [english]
To setup these package repositories, see the documentation regarding the installation of mackerel-agent.